cognee/tasks/translation/translate_content.py

```
@@ -0,0 +1,644 @@
+
+    def __init__(self):
+        if self._client is None:
+            try:
+                from azure.ai.translation.text import TextTranslationClient
+                from azure.core.credentials import AzureKeyCredential
+
+                key = os.getenv("AZURE_TRANSLATE_KEY")
+                endpoint = os.getenv("AZURE_TRANSLATE_ENDPOINT")
+
+                if not all([key, endpoint]):
+                    raise AzureConfigError("Azure Translate credentials (KEY and ENDPOINT) are required.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)